packages/date-time-utilities/src/dateGrid/getDayGrid.ts

```diff
@@ -34,7 +34,7 @@ export const getDayGrid = (options: IDayGridOptions): IDay[][] => {
   let date;
   if (weeksToShow && weeksToShow <= 4) {
     // if showing less than a full month, just use date == navigatedDate
-    date = new Date(navigatedDate.toString());
+    date = new Date(navigatedDate.getFullYear(), navigatedDate.getMonth(), navigatedDate.getDate());
```

Comment:
Please, why don't you do  
```suggestion
    date = new Date(navigatedDate);
```

like in other places?

Reply:
In this case, it's an actual bug fix. See right below in the else case. This is supposed to create dates for the grid starting at midnight, rather than at the same time as whatever the navigatedDate is.